CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [v1.11.2] - 2019-04-18
+
+- #224 Add maintenance window support for Kubernetes- @fatih
```

Comment:
Adding maintenance window support is a new feature rather than a bugfix, so the version number should be 1.12.0

Reply:
It's a sub option of creating a cluster, hence I thought it could be represented by patch. I've checked the changelog and seems like it's a mix of minor and patch upgrades. I'll change it to v1.12.0 👍

Reply:
I understand (and sympathize with!) the confusion, but patch versions are specifically for bug fixes under semver.